Add note to the Tech Specs about removal of archs (#7782)
nginx/kubernetes-ingress: site/content/releases.md

16 Apr 2025

{{< important >}}

> In our upcoming release, `v5.1.0`, the Open Source NGINX Ingress Controller architectures `armv7`, `s390x` & `ppc64le` will no longer be provided.

Reviewer: ```suggestion
From release `v5.1.0` onwards, NGINX Ingress Controller will no longer provide binaries for the `armv7`, `s390x` & `ppc64le` architectures.
```
